thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PerformanceEvaluationUtil.java

```diff
@@ -0,0 +1,224 @@
+package com.linkedin.thirdeye.detector.email.filter;
+
+import com.linkedin.thirdeye.anomalydetection.context.AnomalyFeedback;
+import com.linkedin.thirdeye.constant.AnomalyFeedbackType;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class to evaluate the performance of a list of merged anomalies
+ */
+public class PerformanceEvaluationUtil {
```

Comment:
That's a good catch. I'll rename the class.
This is to provide a class to evaluate precision and recall in ThirdEye. There are two versions of precision and recall calculation in ThirdEye: one is in AlertFilterEvaluationUtils, and the other is in EmailReportGenerator. I want to have a tool to calculate the precision and recall for a past time region, but I cannot use either of them. Therefore, I want to unify the calculation of precision and recall.